serverlessworkflow/modules/ROOT/pages/eventing/consume-producing-events-with-kafka.adoc

```diff
@@ -1,30 +1,23 @@
-Consuming and producing events with Kafka
-=========================================
+= Consuming and producing events using Apache Kafka
 :compat-mode!:
 // Metadata:
 :description: Consuming and producing events with Kafka
 :keywords: kogito, workflow, serverless, events, kafka
 // links
 :kogito_messaging_url: https://docs.jboss.org/kogito/release/latest/html_single/#_kogito_messaging_add_on
 :kogito_kafka_integration_url: https://docs.jboss.org/kogito/release/latest/html_single/#proc-messaging-enabling_kogito-configuring
-:callback_example_url: {kogito_sw_examples_url}/serverless-workflow-callback-quarkus/src/main/resources/application.properties}
-:events_example_url: {kogito_sw_examples_url}/serverless-workflow-events-quarkus/src/main/resources/application.properties}
+:callback_example_url: {kogito_sw_examples_url}/serverless-workflow-callback-quarkus/src/main/resources/application.properties
+:events_example_url: {kogito_sw_examples_url}/serverless-workflow-events-quarkus/src/main/resources/application.properties
 :smallrye_connector_doc_url: {smallrye_messaging_url}/connectors/connectors.html
 :kafka_connector_properties_url: {smallrye_messaging_url}/kafka/kafka.html#_configuration_reference
 
+This document describes how you can configure Serverless Workflow to interact with link:{kafka_doc_url}[Apache Kafka]. Apache Kafka uses an abstraction called topic to classify events. The events are published to a topic, and events are consumed from a topic. 
```

Comment:
One question about styling. I use events without "the" before because Im referring to all posible events, not a concrete set of them

Reply:
Good question: 
I thought we are specifically stating CloudEvents, If you meant the all event, I'll remove the word "the"